examples/interpreter/physl.cpp

```diff
@@ -237,6 +237,12 @@ int main(int argc, char* argv[])
                 << phylanx::execution_tree::dot_tree(code_source_name,
                        snippets.snippets_.back().get_expression_topology())
                 << "\n";
+
+            std::cout << "\n"
+                << phylanx::execution_tree::newick_tree(code_source_name,
+                       snippets.snippets_.back().get_expression_topology())
+                << "\n\n";
+
```

Comment:
Would you see a way to extract the expression-topology only once and reuse it for the printing operations?